subjects/revconcatalternate/README.md

```diff
@@ -0,0 +1,44 @@
+## revconcatalternate
+
+### Instructions
+
+Write a function `RevConcatAlternate()` that receives two slices of `int` as arguments and returns a new slice with alternated values of each slice in reverse order.
```

Comment:
When I ran prettier it complained here. Please run it before asking for the review.